opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -23,35 +23,50 @@ option java_multiple_files = true;
 option java_package = "io.opentelemetry.proto.trace.v1";
 option java_outer_classname = "TraceProto";
 
-// A span represents a single operation within a trace. Spans can be
+// Span represents a single operation within a trace. Spans can be
 // nested to form a trace tree. Spans may also be linked to other spans
-// from the same or different trace. And form graphs. Often, a trace
+// from the same or different trace and form graphs. Often, a trace
 // contains a root span that describes the end-to-end latency, and one
 // or more subspans for its sub-operations. A trace can also contain
 // multiple root spans, or none at all. Spans do not need to be
 // contiguous - there may be gaps or overlaps between spans in a trace.
 //
-// The next id is 15.
+// The next field id is 17.
 message Span {
-  // A unique identifier for a trace. All spans from the same trace share
-  // the same `trace_id`. The ID is a 16-byte array. An ID with all zeroes
-  // is considered invalid.
+  // trace_id is the unique identifier of a trace. All spans from the same trace share
+  // the same `trace_id`.
   //
-  // This field is semantically required. Receiver should generate new
-  // random trace_id if empty or invalid trace_id was received.
+  // This field is semantically required. If 0 or invalid trace_id was received:
+  // - The receiver MAY reject the invalid data and respond with the appropriate error
+  //   code to the sender.
+  // - The receiver MAY accept the invalid data and attempt to correct it.
   //
-  // This field is required.
-  bytes trace_id = 1;
-
-  // A unique identifier for a span within a trace, assigned when the span
-  // is created. The ID is an 8-byte array. An ID with all zeroes is considered
-  // invalid.
+  // The ID is a 16 bytes value represented using two fixed64 fields. An ID with
```

Comment:
`The ID is a 16 bytes value **is** represented using two fixed64 fields.`

Reply:
I think what you suggest is not syntactically correct English. I may be wrong though, I am not a native speaker :-)

Reply:
I would defer to a native speaker :)

Reply:
Language looks good. I'd write "a 16-byte value", no big deal :)
This is making regret that the Go tracer switched from two uint64s to [16]byte.

Reply:
Changed to "a 16-byte value", thanks.

>This is making regret that the Go tracer switched from two uint64s to [16]byte.

Is it too late to switch back? Would be probably nice to use something like `struct { hi,lo uint64 }`.

Reply:
It's not. That's what we had before. :)

Reply:
@jmacd I am going to resolve this, leaving it up to you if you want to consider changing TraceID data type in Go lib.